Update LangChain provider page to Gateway Credits for launch (#5168)
langchain-ai/docs: src/langsmith/llm-gateway-langchain-provider.mdx

---
title: Gateway Credits

> description: Use Gateway Credits to access models without a provider key — just authenticate with LangSmith.

Reviewer: ```suggestion
description: Use Gateway Credits to access models without a provider key, just authenticate with LangSmith.
```

---

<Note>
**Private beta:** The LLM Gateway is in private [beta](/langsmith/release-stages). APIs and features may change as we iterate. Sign up for [the waitlist](https://www.langchain.com/langsmith-llm-gateway-waitlist) to get access.
</Note>

> The `/langchain` provider gives you access to **Gateway Credits**, a way to call models through the LLM Gateway without setting up a provider account or key. Use Gateway Credits to access models without a provider key, just authenticate with LangSmith. With [other providers](/langsmith/llm-gateway#supported-providers), you bring your own account and key. With Gateway Credits, LangChain owns the upstream provider credential, so you can start calling a model immediately with only your [LangSmith API key](/langsmith/create-account-api-key). No [provider secret](/langsmith/llm-gateway-admin-setup#2-add-provider-secrets) setup is required.

Reviewer 2: I would say "gives you access to call models through the LLM Gateway without setting up a provider account or key. Gateway Credits are required to access models in this way."

> The `/langchain` provider gives you access to **Gateway Credits**, a way to call models through the LLM Gateway without setting up a provider account or key. Use Gateway Credits to access models without a provider key, just authenticate with LangSmith. With the [other providers](/langsmith/llm-gateway#supported-providers), you bring your own account and key. With Gateway Credits, LangChain owns the upstream provider credential, so you can start calling a model immediately with only your [LangSmith API key](/langsmith/create-account-api-key). No [provider secret](/langsmith/llm-gateway-admin-setup#2-add-provider-secrets) setup is required.

Reviewer 3: ```suggestion
The `/langchain` provider gives you access to **Gateway Credits**, a way to call models through the LLM Gateway without setting up a provider account or key. Use Gateway Credits to access models without a provider key, just authenticate with LangSmith. With [other providers](/langsmith/llm-gateway#supported-providers), you bring your own account and key. With Gateway Credits, LangChain owns the upstream provider credential, so you can start calling a model immediately with only your [LangSmith API key](/langsmith/create-account-api-key). No [provider secret](/langsmith/llm-gateway-admin-setup#2-add-provider-secrets) setup is required.
```

> The `/langchain` provider gives you access to **Gateway Credits** — a way to call models through the LLM Gateway without setting up a provider account or key. Use Gateway Credits to access models without a provider key — just authenticate with LangSmith. With the [other providers](/langsmith/llm-gateway#supported-providers), you bring your own account and key. With Gateway Credits, LangChain owns the upstream provider credential, so you can start calling a model immediately with only your [LangSmith API key](/langsmith/create-account-api-key). No [provider secret](/langsmith/llm-gateway-admin-setup#2-add-provider-secrets) setup is required.

Reviewer: ```suggestion
The `/langchain` provider gives you access to **Gateway Credits**, a way to call models through the LLM Gateway without setting up a provider account or key. Use Gateway Credits to access models without a provider key, just authenticate with LangSmith. With the [other providers](/langsmith/llm-gateway#supported-providers), you bring your own account and key. With Gateway Credits, LangChain owns the upstream provider credential, so you can start calling a model immediately with only your [LangSmith API key](/langsmith/create-account-api-key). No [provider secret](/langsmith/llm-gateway-admin-setup#2-add-provider-secrets) setup is required.
```

## Available models

The provider is OpenAI-compatible and advertises branded model IDs. Select a model by name in the request body, and the gateway routes it to the upstream model. Model IDs are case-insensitive.

> | Model ID | Description |

Reviewer 3: @kyrawong are these the correct model variants listed?

Reviewer 4: @bvs-langchain this needs to be changed to "Fireworks-compatible" right?

Reviewer 2: https://fireworks.ai/models/fireworks/kimi-k3
https://fireworks.ai/models/fireworks/kimi-k2p6

these are the models. IDs are `moonshotai/Kimi-K2.6` and `moonshotai/Kimi-K3`

| --- | --- |
| `POST /langchain/v1/chat/completions` | Chat completions, including streaming. |
| `POST /langchain/v1/responses` | OpenAI Responses API. |
| `GET /langchain/v1/models` | Lists the available model IDs. |

Any other path returns `404`. A request for a model that isn't advertised also returns `404` with a message listing the available model IDs.

## Pricing

> Gateway Credits are denominated in **LangChain Credit Units (LCUs)** at **$1.50 per LCU**. Each model call consumes LCUs based on token usage. See [langchain.com/pricing](https://www.langchain.com/pricing) for current rates and plan details.

Reviewer 3: ```suggestion
Gateway Credits are denominated in **LangChain Credit Units (LCUs)** at **$1.50 per LCU**. Each model call consumes LCUs based on token usage. For current rates and plan details, refer to [langchain.com/pricing](https://www.langchain.com/pricing).
```

> Standard gateway [spend policies](/langsmith/llm-gateway-spend-policies) still apply, so you can add per-key or per-user limits on top of your Gateway Credit balance.

Reviewer 2: If your plan supports LLM Gateway (link to pricing page)

> Standard gateway [spend policies](/langsmith/llm-gateway-spend-policies) still apply, so you can add per-key or per-user limits on top of your Gateway Credit balance.

Reviewer 2: would say "so your per-key, per-user, and other scoped policies apply to `/langchain` usage"

## Tracing

> Like all gateway traffic, Gateway Credit calls are traced to LangSmith. See [Traces, Engine, and access control](/langsmith/llm-gateway-access) for where traces land and how to control access to them.

Reviewer 3: ```suggestion
Like all gateway traffic, Gateway Credit calls are traced to LangSmith. For where traces land and how to control access to them, refer to [Traces, Engine, and access control](/langsmith/llm-gateway-access).
```

> Like all gateway traffic, Gateway Credit calls are traced to LangSmith. For where traces land and how to control access to them, refer to [Traces, Engine, and access control](/langsmith/llm-gateway-access).

Reviewer 2: instead of "Gateway Credit calls" would say "calls to `/langchain`" which is the technical description
